### PR TITLE
added event type in Event class #650

### PR DESCRIPTION
**Pull request by Paritosh-Anand**
This change is [Reviewable](https://reviewable.io/reviews/docker-java/docker-java/651)

### CODE REVIEW

**Comment by Paritosh-Anand** [comment]
CI check is failing because of older docker api version i.e. 1.20

Need to change this to 1.24, can any of the moderators help here?

```
docker version
Client:
 Version:      1.8.2
 API version:  1.20
 Go version:   go1.4.2
 Git commit:   0a8c2e3
 Built:        Wed Oct  7 17:48:28 UTC 2015
 OS/Arch:      linux/amd64

Server:
 Version:      1.8.2
 API version:  1.20
 Go version:   go1.4.2
 Git commit:   0a8c2e3
 Built:        Wed Oct  7 17:48:28 UTC 2015
 OS/Arch:      linux/amd64
```

File: src/main/java/com/github/dockerjava/api/model/Event.java [hunk 1]
```diff
@@ -23,6 +23,9 @@
     @JsonIgnoreProperties
     private Node node;
 
+    @JsonProperty("Type")
+    private String type;
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/model/Event.java]: Add javadoc`@since RAP`.. (see examples in existing methods)

**Comment by KostyaSha** [comment]
Add in netty and jersey test classes check for this field in conditional situation like in https://github.com/docker-java/docker-java/pull/642/files#r72670628

**Comment by KostyaSha** [comment]
I have pre-defined reply:

- Please provide information when this field appeared (you should compare API versions https://docs.docker.com/engine/reference/api/docker_remote_api/ ) 
- Please read https://github.com/docker-java/docker-java/blob/master/docs/devel.adoc
- Please ensure that option is added to jaxrs and netty implementation.

File: src/main/java/com/github/dockerjava/api/model/Event.java [hunk 2]
```diff
@@ -94,6 +101,11 @@ public Node getNode() {
         return node;
     }
 
+    @JsonProperty("Type")
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/model/Event.java]: you don't need it, because you put it on field

Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/model/Event.java]: `@CheckForNull`

**Comment by Paritosh-Anand** [comment]
@KostyaSha I have made the required changes as you suggested. Still the build is failing, it seems that test suite is not able to connect to Docker host. Is there something else which I am missing here ?

**Comment by KostyaSha** [comment]
will check as soon as find free time

**Comment by KostyaSha** [comment]
Could you provide JSON of some event from real system to make test?

**Comment by KostyaSha** [comment]
nevermind, picked from docs

**Comment by Paritosh-Anand** [comment]
Great !!

**Pull request closed without merging**